apps/script/generateSharedConstants.rb

```diff
@@ -57,7 +57,7 @@ def generate_multiple_constants(shared_const_names, *options)
 def parse_raw(raw)
   if raw.is_a?(OpenStruct)
     raw.marshal_dump
-  elsif raw.is_a?(Hash) || raw.is_a?(Array)
+  elsif raw.is_a?(Hash) || raw.is_a?(Array) || raw.is_a?(Integer)
```

Comment:
no longer needed?

Reply:
yuck - yes, no longer needed. Thanks for catching.